yarn.lock

```diff
@@ -68,6 +68,15 @@
     jsesc "^2.5.1"
     source-map "^0.5.0"
 
+"@babel/generator@^7.12.0":
```

Comment:
Nit: it seems that many packages in the lockfile are duplicated. Maybe `yarn yarn-deduplicate` would help reducing the size of `node_modules` for your local development.

Reply:
@nicolo-ribaudo - Ya, good point! I'll do that on master (we are queuing up a major release over there for some dependency updates).